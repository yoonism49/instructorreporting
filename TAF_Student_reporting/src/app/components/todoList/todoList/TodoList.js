// @flow weak
import React        from 'react';
import PropTypes    from 'prop-types';

const TodoList = ({
  children
}) => (
  <div className="task-content">
    <ul className="task-list">
      {children}
    </ul>
  </div>
);

TodoList.propTypes = {
  children: PropTypes.node.isRequired
};

export default TodoList;
