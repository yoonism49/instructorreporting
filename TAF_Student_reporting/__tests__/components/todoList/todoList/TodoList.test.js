
'use strict';

import React              from 'react';
import TodoList           from '../../../../src/app/components/todoList/todoList/TodoList';
import renderer           from 'react-test-renderer'; // needed both for snpashot testing but also to prevent errors from enzyme

describe('TodoList component', () => {
  it('renders as expected', () => {
    const component = renderer.create(
      <div>
        <TodoList>
          <p>todo list children</p>
        </TodoList>
      </div>
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
