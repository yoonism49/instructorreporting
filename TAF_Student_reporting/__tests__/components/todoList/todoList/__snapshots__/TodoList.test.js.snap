// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoList component renders as expected 1`] = `
<div>
  <div
    className="task-content"
  >
    <ul
      className="task-list"
    >
      <p>
        todo list children
      </p>
    </ul>
  </div>
</div>
`;
